ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -606,9 +507,15 @@ func (s *LifeCycleServiceServer) Init(ctx context.Context, in *pb.InitRequest) (
 		}
 	}
 
-	// Preconfigure the FXP
-	DelFxpRules(s.p4rtbin)
-	AddFxpRules(s.p4rtbin)
+	vfMacList, err := utils.GetVfMacList()
+
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Unable to reach the IMC %v", err)
+	}
+
+	// Preconfigure the FXP with point-to-point rules between host VFs
+	p4rtclient.CreateDelPointToPointVFRules(s.p4rtbin, vfMacList)
```

Comment:
nit: maybe use names like `CreatePointToPointVFRules` and `DeletePointToPointVFRules` instead?

Reply:
Functions renamed as described in the comment.